packages/suite-desktop-core/src/libs/sentry.ts

```diff
@@ -20,18 +20,12 @@ export const initSentry = ({ mainThreadEmitter, store }: InitSentryParams) => {
     });
 
     const transportOptions = {
-        beforeSend: () => {
-            if (store.getTorSettings().running && torStatus !== TorStatus.Enabled) {
-                // If Tor is enabled but not running, don't send the event but put it in a queue.
-                // Queue can be inspected in @trezor/suite-desktop/sentry/queue folder.
-                return 'queue';
-            }
-
-            return 'send';
-        },
+        // If Tor is enabled but not running, don't send the event but put it in a queue.
+        // Queue can be inspected in @trezor/suite-desktop/sentry/queue folder.
+        shouldSend: () => !(store.getTorSettings().running && torStatus !== TorStatus.Enabled),
```

Comment:
API change, see [documentation](https://github.com/trezor/trezor-suite/pull/15383/commits/ac51816d9f33f663c90df0c70deefdcebc870bf9).

Reply:
Sorry for troubles I didn't expect it could affect desktop and TS won't even complain